developer-docs-site/docs/nodes/node-health-checker.md

```diff
@@ -121,7 +121,7 @@ Next, download these configuration YAML files into the `/etc/nhc` folder in your
 ```
 mkdir /etc/nhc
 cd /etc/nhc
-configs=(single_node_validator devnet_fullnode ait2_validator); for c in ${configs[@]}; do wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/ecosystem/node-checker/configurations/$c.yaml; done
+configs=(single_node_validator devnet_fullnode ait3_validator); for c in ${configs[@]}; do sudo wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/ecosystem/node-checker/configuration_examples/$c.yaml; done
```

Comment:
This is a good catch!  Most probably do not have permissions to `/etc/nhc`.  Though, the permissions will likely need to be changed to allow for it.

Let's instead do a `sudo chown $USER /etc/nhc` prior to this command, which will then allow the user to have access to these files without running `sudo` later.

Reply:
Great point. I would guess that changing ownership of the file after and not elevating the wget would be a better practice from a security standpoint.